app/uk/gov/hmrc/partnershipidentification/testonly/RegisterWithMultipleIdentifiersStubController.scala

```diff
@@ -24,7 +24,7 @@ import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 @Singleton
 class RegisterWithMultipleIdentifiersStubController @Inject()(controllerComponents: ControllerComponents) extends BackendController(controllerComponents) {
 
-  val registerWithMultipleIdentifiers: Action[AnyContent] = Action {
+  def registerWithMultipleIdentifiers(regime: String): Action[AnyContent] = Action {
```

Comment:
regime is warning that it's not being used, but I assume it's there because it's needed for the route to work?

Reply:
Yes, thats the reason